### PR TITLE
Stories/1331 add reading time indicator

### LINKED ISSUE #1331: Add reading time indicator
### Description

We should implement a reading time indicator that calculates the expected reading time for individual blog posts. This information should be displayed on yoast.com. Later this might also be implemented as a feature on Premium.

For details on the formula see: https://github.com/Yoast/yoast.com/issues/1395

### PR DESCRIPTION
**Pull request by manuelaugustin**
## Summary

This PR can be summarized in the following changelog entry: adds a researcher to calculate the reading time for a given paper.

## Relevant technical choices:

* The reading time is always rounded up to the next full minute (e.g. 0.2 -> 1) to provide a conservative measure.
* Currently this assessment uses a basic formula based on text length and the number of images. The original reading time formula (see https://github.com/Yoast/yoast.com/issues/1395) also included code tags. With the current text parsing method we cannot extract this information. This functionality will be added later when we have the relevant text parsing ability.
* The reading time indicator is initially intended for use on yoast.com only. It currently is not meant to be accessed from within the content analysis of the plugin.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` to check whether none of the unit tests fail.
* Create an additional test using a text of your choice. Check whether it returns a meaningful reading time.

Fixes #1331

### CODE REVIEW

**Comment by igorschoester** [comment]
CR 👍

File: src/researches/getReadingTime.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let wordCountInText = require( "./wordCountInText.js" );
+let imageCount = require( "./imageCountInText.js" );
+
+module.exports = function( paper ) {
+	let numberOfWords = wordCountInText( paper );
+	let numberOfImages = imageCount( paper );
+
+	return Math.ceil( ( numberOfWords / 200 ) + ( numberOfImages * 0.2 ) );
```
Comment (Xyfi) [inline src/researches/getReadingTime.js]: A specification of how this calculation came to be referencing the technical choices would be nice.

**Comment by afercia** [comment]
CR 👍 
Applied some CS to the spec file.

**Comment by afercia** [comment]
Acceptance OK 👍 
Tested with different kind of content including HTML, 5 or more images, tables, etc.

**Pull request merged**